README.md

```diff
@@ -1 +1,26 @@
-# InterviewQuestion
\ No newline at end of file
+# Front End interview
+
+We have a very basic sign-up form written in React that we'd like you to flesh out.  We have the following fields…
+1. First name
+2. Last name
+3. Phone number
+4. User name
+5. Password
+6. Date of Birth
+
+...and a submit button.
+
+We'd like you to do the following:
+1. Connect this to our backend service that will either accept or reject the signup: https://react-interview-question-backend-evdphnararfqcvcm.canadacentral-01.azurewebsites.net
+    - The service expects a json body that will conforms to the spec described here: <insert swagger url>
```

Comment:
Could we add a point which says
if the services accepts. Display whats added to the services and dont show the form

but the service does not accept display error

Reply:
My thinking is that it be user focused.  Don't show the user the error that comes from the service, but show information to the user that helps them identify the problem (if the interviewee doesn't just validate in the UI what is wrong before allowing them to submit--which IMO would be a better user-centered approach).

Reply:
That works too!!